proxy/src/telemetry/metrics/mod.rs

```diff
@@ -22,8 +25,10 @@ use control::pb::proxy::telemetry::{
     StreamSummary,
     TransportSummary,
 };
-use ctx;
-use telemetry::event::Event;
+use telemetry::event::{Event};
+
+mod latency;
+use self::latency::*;
```

Comment:
I know some places already have star imports, but I'd rather promote that we don't use them, as it improves readability for Future Us.